content/opt/run

```diff
@@ -219,5 +212,5 @@ echo "==================================================================="
 touch ${initfile}
 fi
 
-echo "Starting Supervisor.  You can safely CTRL-C and the container will continue to run with or without the -d (daemon) option"
+echo "Starting Supervisor."
```

Comment:
Just tested it out and it does appear to be the case.  To detach from an interactive session, you would need to use sequence of CTRL + P and CTRL + Q sequence.  

Unless there's a reason to start a container in interactive mode, I think the general recommendation is to run with the -d daemon flag or since this container uses supervisor to run the services, you can run it without the -d daemon flag and just escape out